pkg/monitoring/prometheusagent/types.go

```diff
@@ -13,8 +13,7 @@ const (
 	// servicePriorityLabel is the label used to determine the priority of a service.
 	servicePriorityLabel string = "giantswarm.io/service-priority"
 
-	remoteWriteEndpointTemplateURL = "https://prometheus.%s/%s/api/v1/write"
-	remoteWriteName                = "prometheus-meta-operator"
+	remoteWriteEndpointTemplateURL = "https://%s.%s/%s/api/v1/%s"
```

Comment:
I think most of the %s should be hardcoded as mimir will always be in the same api

Reply:
Updated it